src/groovy/grails/plugin/springsecurity/web/authentication/rememberme/GormPersistentTokenRepository.groovy

```diff
@@ -60,7 +60,10 @@ class GormPersistentTokenRepository implements PersistentTokenRepository, Grails
 		def persistentToken
 		def clazz = lookupDomainClass()
 		if (clazz) {
-			persistentToken = clazz.get(seriesId)
+			// join an existing transaction if one is active
+			clazz.withTransaction { status ->
```

Comment:
Wouldn't it make more sense to use withNewSession instead? Not all GORM implementations support transactions but withNewSessions should work for all.